src/views/hoc/hoc.jsx

```diff
@@ -30,55 +30,49 @@ var Hoc = React.createClass({
             <div className="hoc">
                 <TitleBanner className={this.state.bgClass}>
                     <h1>
-                        <FormattedMessage
-                            id='hoc.title'
-                            defaultMessage={'Get Creative with Coding'} />
+                        <FormattedMessage id='hoc.title' />
                     </h1>
                     <p>
-                        <FormattedMessage
-                            id='hoc.subTitle'
-                            defaultMessage={
-                                'With Scratch, you can program your own stories, games, and animations — ' +
-                                'and share them online.'
-                            } />
+                        <FormattedMessage id='hoc.subTitle' />
                     </p>
 
-                    <section>
+                    <section className="section title-banner-section">
                         <FlexRow>
-                            <div className="card">
+                            <div className="flex-row-card">
                                 <a href="/projects/editor/?tip_bar=name">
-                                    <div className="card-info" onMouseEnter={this.onCardEnter.bind(this, 'name-bg')}>
-                                        <img src="/images/hoc2015/name-tutorial.jpg" alt="" />
-                                        <Button>
-                                            <FormattedMessage
-                                                id='general.tipsAnimateYourNameTitle'
-                                                defaultMessage={'Animate Your Name'} />
+                                    <div className="flex-row-card-info"
+                                         onMouseEnter={this.onCardEnter.bind(this, 'name-bg')}>
+                                        <img className="img card-info"
```

Comment:
This actually was an attempt at following that example, which has a default button styling (i.e. here it would be an image styling) with mod, which is `card-info`. I guess it could be called `mod-flex-row-card-info`, though. Seems like if we're going to use `img` as a class, we shouldn't also put it on the mod, though.